tests/RobotFramework/tests/azure/azure_telemetry.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-az subscribes the topics that are configured as az.topics
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Custom Teardown
+
+Test Tags           theme:mqtt    theme:az
+
+
+*** Test Cases ***
+
+Publish measurements to te measurement topic
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main///m/ '{"temperature": 10}'
+    Should Have MQTT Messages    az/messages/events/#   message_contains="temperature"    date_from=${timestamp}   minimum=1    maximum=1
+
+Publish measurements to te measurement topic with measurement type
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main///m/test-type '{"pressure": 10}'    
+    Should Have MQTT Messages    az/messages/events/#   message_contains="pressure"    date_from=${timestamp}   minimum=1    maximum=1
+
+Publish measurements to te measurement topic without measurement type
```

Comment:
This test has no diff from the first test `Publish measurements to te measurement topic`. I suggest to keep this and remove the other one.